BLT/BLT/Focus Mode View/FocusViewController.swift

```diff
@@ -70,7 +77,14 @@ class FocusViewController: UIViewController, FocusTimerDelegate {
 //            endFocusModeButton.isEnabled = true
 //            endFocusModeButton.isHidden = false
 //        }
-
+        popup = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "sbPopUpID") as! FMPopUpViewController
```

Comment:
Refactor to an if let statement to avoid a force cast using as!
```Swift
if let temp = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "sbPopUpID") as? FMPopUpViewController {
   self.popup = temp
}
```